charts/ckan/templates/ckan/configmap.yaml

```diff
@@ -63,174 +100,245 @@ data:
     ckan.auth.create_user_via_api = true
     ckan.auth.create_user_via_web = true
     ckan.auth.roles_that_cascade_to_sub_groups = admin
-    
+    ckan.auth.public_user_details = true
+    ckan.auth.public_activity_stream_detail = false
+    ckan.auth.allow_dataset_collaborators = false
+    ckan.auth.allow_admin_collaborators = false
+    ckan.auth.allow_collaborators_to_change_owner_org = false
+    ckan.auth.create_default_api_keys = false
+    ckan.auth.login_view = user.login
+    ckan.auth.reveal_private_datasets = false
+    ckan.auth.enable_cookie_auth_in_api = true
+    ckan.auth.route_after_login = dashboard.datasets
+
     ## User Account Creation Setting
-    ckan.valid_email_regexes = .gov.uk$ .nhs.uk$ .nhs.net$ .ac.uk$ .os.uk$ .mod.uk$ .police.uk$ .bl.uk$
-    
-    ## CORS Settings
-    
-    # If cors.origin_allow_all is true, all origins are allowed.
-    # If false, the cors.origin_whitelist is used.
-    ckan.cors.origin_allow_all = true
-    # cors.origin_whitelist is a space separated list of allowed domains.
-    # ckan.cors.origin_whitelist = http://example1.com http://example2.com
-    
-    
-    ## Plugins Settings
-    
-    # Note: Add ``datastore`` to enable the CKAN DataStore
-    #       Add ``datapusher`` to enable DataPusher
-    #       Add ``resource_proxy`` to enable resorce proxying and get around the
-    #       same origin policy
-    
-    ckan.plugins = datagovuk_publisher_form datagovuk dcat harvest ckan_harvester dcat_rdf_harvester dcat_json_harvester dcat_json_interface spatial_metadata spatial_query spatial_harvest_metadata_api gemini_csw_harvester gemini_waf_harvester gemini_doc_harvester inventory_harvester
-    
-    # These are marked as legacy harvesters
-    # gemini_csw_harvester gemini_doc_harvester gemini_waf_harvester
-    
-    # This needs fixing
-    # inventory_harvester
-    
-    
+    ; ckan.valid_email_regexes = .gov.uk$ .nhs.uk$ .nhs.net$ .ac.uk$ .os.uk$ .mod.uk$ .police.uk$ .bl.uk$
+
+    ## CSRF Protection #############################################################
+    WTF_CSRF_ENABLED = true
+    WTF_CSRF_CHECK_DEFAULT = true
+    WTF_CSRF_SECRET_KEY = _e6qkoYldyll9U3zfy5n6rg-JxCPY4DUfy5LHu3qnUo
```

Comment:
If this is actually used then you'll probably need to change it now 😅 If it's a dummy value or unused, then it's a good idea to change it to something that's more obviously a dummy/placeholder value so that it's clear that it's ok.

Reply:
I'll probably auto generate this in the init